BostonData/lambda_function/get_alerts_intent.py

```diff
@@ -102,9 +103,12 @@ def get_alerts():
         alerts[services[i]] = service_info[i]
     # get alert header, if any (this is something like "Winter Storm warning")
     header = ""
-    if soup.find(class_ = "t--upper t--sans lh--000 t--cb") != None:
+    if soup.find(class_ = "t--upper t--sans lh--000 t--cb") != None: # no guarantee that this tag will always be the header
         header += soup.find(class_ = "t--upper t--sans lh--000 t--cb").text + '. '
         header += soup.find(class_ = "str str--r m-v300").text + '. ' 
         header += soup.find(class_ = "t--sans t--cb lh--000 m-b500").text + ' '
+
+    if soup.find(class_ = "t--sans t--ob lh--000 m-b500") != None: # another possible header tag, again this might change 
+        header += soup.find(class_ = "t--sans t--ob lh--000 m-b500").text
```

Comment:
Maybe define these strings as constants at the beginning of the get_alerts() function? Might save some time and energy if the site does end up changing. Also, then the names of the strings could help readability, like if they're called "WINTER_WARNING_ALERT_TAG", "PARKING_INFO_ALERT_TAG", etc.